lib/imapoptions

```diff
@@ -172,6 +172,14 @@ are listed with ``<none>''.
 /* Defaults to enabled.  If disabled, disallows the use of the SETACL
    command at all via IMAP. */
 
+{ "allowspecialusesubfolder", 0, SWITCH, "3.0.14" }
```

Comment:
Don't hardcode version numbers here, just put "UNRELEASED" and the real version will be put in when a release is made.

```suggestion
{ "allowspecialusesubfolder", 0, SWITCH, "UNRELEASED" }
```